dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/bnd.bnd

```diff
@@ -42,3 +42,11 @@ javac.target: 1.8
   com.ibm.wsspi.org.osgi.service.component,\
   com.ibm.wsspi.org.osgi.service.component.annotations,\
   javax.activation:activation;version=1.1
+
+-testpath: \
```

Comment:
There should also be a .classpath change to add the test directory as a project source path.